new_file.txt

```diff
@@ -1 +1,3 @@
-hello git
\ No newline at end of file
+hello world
+
+hello sadid
```

Comment:
capitalze the letter S

Reply:
You are right, let me make the fix.

Reply:
@Shadid-dev  I have made the change and the commit is https://github.com/dividedbyzero360/git_demo/pull/1/commits/4d991d3e5da85d2d1e01e7856c37c5991becb921